kong/db/strategies/cassandra/connector.lua

```diff
@@ -309,4 +309,87 @@ function CassandraConnector:truncate_table(table_name)
 end
 
 
+function CassandraConnector:setup_locks(default_ttl)
+  local ok, err = self:connect()
+  if not ok then
+    return nil, err
+  end
+
+  local cql = string.format([[
+    CREATE TABLE IF NOT EXISTS locks(
+      key text PRIMARY KEY,
+      owner text
+    ) WITH default_time_to_live = %d
+  ]], default_ttl)
```

Comment:
Shouldn't the above be in a migration? Not as much for this instance as well as creating a precedent.

Unless the intent is to use this lock, during migrations?

Reply:
> Unless the intent is to use this lock, during migrations?

Yes indeed